managed-ledger/src/main/proto/MLDataFormats.proto

```diff
@@ -71,6 +71,9 @@ message PositionInfo {
 
     // Store which index in the batch message has been deleted
     repeated BatchedEntryDeletionIndexInfo batchedEntryDeletionIndexInfo = 5;
+
+    // serialized individualDeletedMessages
+    optional bytes serializedIndividualDeletedMessages = 6;
```

Comment:
Instead of having a byte array here, I think it would be better to have instead a way to encode the range set in protobuf directly

Reply:
I tried to think about it but I don't see a clean way to get an internal structure of `BitSet` or internal reference which we can use to encode to protobuf. So, only way, I can see is to seralize BitSet to byte array and use the same to deserialize, It also gives a flexibility if we want to make range-set pluggable which can have a different implementation and that range set serialization doesn't have to depend on any specific protobuf format.
but please let me know if you have any recommendation which can allow us to encode it directly in protobuf

Reply:
My main concern here is that we're mixing 2 serialization systems together and therefore we need to pay attention when there are changes to that format. That and that Java serialization is really bad in general.

Reply:
> That and that Java serialization is really bad in general.

I agree. let me address it.

Reply:
@merlimat removed the serialization and enocded to protobuf.